src/LinkedListVisualizer/LinkedListVisualizer.jsx

```diff
@@ -0,0 +1,497 @@
+import React, {Component} from 'react';
+import {useTransition, animated} from 'react-spring';
+import Node from './Node/Node';
+import './LinkedListVisualizer.css';
+import Pointer from './Node/Pointer';
+import InputBox from '../Components/InputBox';
+
+export default class LinkedListVisualizer extends Component {
+    constructor(){
+        super();
+        this.state = {
+            numberOfNodes: 0,
+            grid: [],
+            inputGrid: [ { data: "" }],
+            cycleTo: 0,
+            cycleToHolder: 0,
+            cyclePresent: false,
+            boxWidth: 0,
```

Comment:
I don't think this needs to be part of your state; you never set it to anything.